server.js

```diff
@@ -10,7 +11,19 @@ app.use(bodyparser.json());
 app.use(cors());
 
 const dbURL = 'mongodb://localhost/todo';
+// for CORS
+//CORS middleware
+var corsMiddleware = function(req, res, next) {
```

Comment:
was there a cors error, due to which you added this? Because I already imported the "cors" package which handles all this. Unless there was an error, you should remove these lines.